server/index.js

```diff
@@ -36,6 +36,33 @@ app.get('/api/products', (req, res, next) => {
       console.error(err);
       res.status(500).json({ error: 'an error hs occurred' });
     });
+
+});
+
+app.get('/api/products/:productId', (req, res, next) => {
+  const { productId } = req.params;
+  parseInt(productId);
```

Comment:
Careful. `parseInt` doesn't update the variable, it returns the new integer value.